### DISCUSSION TITLE
Is there any block/warp/instruction shape list for SIMT documented like other op code class?

### DISCUSSION THREAD

**Discussion by TigerYang414** [NVIDIA/cutlass#560] [Q&A]
Other opcode shapes is listed with details in doc:https://github.com/NVIDIA/cutlass/blob/master/media/docs/functionality.md

Could we add SIMT related shapes in it?

**Comment by hwu36** [comment]
there are many more available simt tile sizes than tensor op.  [Here](https://github.com/NVIDIA/cutlass/blob/master/tools/library/scripts/generator.py#L3654-L3664) are some plausible ones for sm80 sgemm.  

for example, in 

`TileDescription([256, 128, 8], 5, [4, 2, 1], math_inst, min_cc, max_cc),`

[256, 128, 8] is the threadblock tile size
[4, 2, 1] is warp count size.  warp tile size is then 256/4 = 32, 128/2=64, 8/1=8
5 is stage number
